### DISCUSSION TITLE
2 table

### DISCUSSION THREAD

**Discussion by theo18112** [codeigniter4/shield#1080] [Q&A]
how to setup for 2 login with different database table
1 admin login
2 member/user login

**Comment by kenjis** [comment]
Shield is designed to have only one user table.
If you want to have two user tables, you need to implement it by yourself.

**Comment by theo18112** [comment]
> Shield is designed to have only one user table. If you want to have two user tables, you need to implement it by yourself.

but i need to extend shiled controller?

**Reply by kenjis** [reply]
You need to create everything you need.

But what do you want exactly? Do you really need to have two user tables?
If you really need two tables, why?

**Comment by theo18112** [comment]
> You need to create everything you need.
> 
> But what do you want exactly? Do you really need to have two user tables? If you really need two tables, why?

yes i need  table for administrators and members login

**Reply by kenjis** [reply]
Then first create tables that you need.
https://github.com/codeigniter4/shield/blob/341ba47106387edc4e2b55531f21a59579d79746/src/Config/Auth.php#L409-L417

It seems at least you need new `users` table for admins.
